docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+# Glossary
+
+- [API Review](#api-review)
+- [Code Freeze](#code-freeze)
+- [Exceptions](#exceptions)
+- [Enhancements Freeze](#enhancements-freeze)
+- [Kubernetes Enhancements Proposal (KEP)](#kubernetes-enhancement-proposal-kep)
+- [Production Readiness Review (PRR)](#production-readiness-review-prr)
+
+### API Review
+The API review process is intended to maintain logical and functional integrity of the
+ API over time, the consistency of user experience and the ability of previously
+ written tools to function with new APIs.
+
+If an enhancement is considered to be changing the Kubernetes core API in any way,
+ a mandatory API review is required. If you are unsure, please do have a look at
+ the document linked below. An Enhancement owner should proactively request an
```

Comment:
strike: "If you are unsure, please do have a look at the document linked below."